README.md

```diff
@@ -42,34 +42,53 @@ sudo ninja -C build install
 
 Launch wob in a terminal, enter a value (positive integer), press return.
 
-```
+```shell script
 wob
 ```
 
 ### General case
 
 You may manage a bar for audio volume, backlight intensity, or whatever, using a named pipe. Create a named pipe, e.g. /tmp/wobpipe, on your filesystem using.
 
-```
+```shell script
 mkfifo /tmp/wobpipe
 ```
 
 Connect the named pipe to the standard input of an wob instance.
 
-```
+```shell script
 tail -f /tmp/wobpipe | wob
 ```
 
 Set up your environment so that after updating audio volume, backlight intensity, or whatever, to a new value like 43, it writes that value into the pipe:
 
-```
+```shell script
 echo 43 > /tmp/wobpipe
 ```
 
+Wob will accept values from 0 to `--max` (default 100) and stop running on non-valid input.
```

Comment:
I am not a native speaker, but wouldn't it be better to write `(by default 100)`? And, `and stop running` could be exit? It sound better to me